examples/medplum-provider/src/pages/encounter/EncounterModal.tsx

```diff
@@ -131,35 +132,7 @@ export const EncounterModal = (): JSX.Element => {
             .
           </Text>
 
-          <Select
-            label="Template Plan definition"
-            placeholder="Initial encounter template"
-            data={['Initial encounter template', 'Follow-up template']}
-            value=""
-            styles={{
-              label: {
-                marginBottom: '0.5rem',
-              },
-            }}
-          />
-
-          <Text size="sm" fw={500} mt="md" mb="xs">
-            Initial encounter plan (these tasks to be added to encounter):
-          </Text>
-
-          <Stack gap="xs">
-            <Card padding="sm" radius="sm" withBorder>
-              <Text size="sm" style={{ textTransform: 'uppercase' }}>
-                CHECK INSURANCE STATUS
-              </Text>
-            </Card>
-
-            <Card padding="sm" radius="sm" withBorder>
-              <Text size="sm" style={{ textTransform: 'uppercase' }}>
-                ORDER BASELINE LAB TESTS
-              </Text>
-            </Card>
-          </Stack>
+          <ResourceInput name="foo" resourceType="PlanDefinition" />
```

Comment:
I assume we need to either use a proper `name` here, or add an `onChange` handler?